src/theme.tsx

```diff
@@ -849,81 +890,61 @@ export const theme = createTheme({
 					border: `1px solid ${color.border.palette.neutral.value} !important`,
 					borderRadius: '50px',
 					width: 'fit-content',
+					gap: '2px',
 				},
 			},
 		},
 		MuiToggleButton: {
+			defaultProps: {
+				disableRipple: true, // needs to be disabled to target with :only-child (see below)
+			},
 			styleOverrides: {
 				root: ({ theme }) => ({
 					textTransform: 'none',
 					border: `none !important`,
-					borderRadius: '50px !important',
+					borderRadius: `${shape.border_radius.full.value}px !important`,
+					font: typography.body.md.shorthand.value,
+					gap: theme.spacing(2),
+					height: '31px', // Fix height to prevent variations due to different icon sizes
 					margin: '2px',
-					padding: '8px',
+					paddingTop: '5px', // 5px = button padding - margin - border width
+					paddingBottom: '5px',
+					paddingLeft: theme.spacing(3),
+					paddingRight: theme.spacing(3),
+					'&:has(> svg:only-child)': {
+						// Fix width when SVG is the only child
+						// FIXME this also targets buttons that have an icon + some text node.
+						//  Text needs to be wrapped in a HTML element to prevent the fixed width.
```

Comment:
We can fix this with a data attribute or className, right?

Reply:
that would be ideal, but can we add that in a global way on a MUI component? That would be "if this prop is set and this prop is empty, add a class"